src/http.zig

```diff
@@ -607,15 +629,32 @@ fn NewHTTPContext(comptime ssl: bool) type {
             );
         }
 
-        pub fn init(this: *@This()) !void {
+        pub fn initWithThreadOpts(this: *@This(), init_opts: *const HTTPThread.InitOpts) InitError!void {
+            if (!comptime ssl) {
+                @compileError("ssl only");
+            }
+            var opts: uws.us_bun_socket_context_options_t = .{
+                .ca = if (init_opts.ca.len > 0) @ptrCast(init_opts.ca) else null,
+                .ca_count = @intCast(init_opts.ca.len),
+                .ca_file_name = if (init_opts.abs_ca_file_name.len > 0) init_opts.abs_ca_file_name else null,
+
+                // TODO: is this needed?
+                .request_cert = 1,
```

Comment:
This was set to `1` in the original function. Keeping it, but unsure if it's needed

Reply:
`request_cert` will guarantee that we set `SSL_VERIFY_PEER` on BoringSSL so is required